warehouse/query-core/src/main/java/datawave/query/discovery/DiscoveryTransformer.java

```diff
@@ -17,15 +13,17 @@
 import datawave.webservice.query.logic.BaseQueryLogicTransformer;
 import datawave.webservice.query.result.event.EventBase;
 import datawave.webservice.query.result.event.FieldBase;
-import datawave.webservice.query.result.event.ResponseObjectFactory;
 import datawave.webservice.query.result.event.Metadata;
+import datawave.webservice.query.result.event.ResponseObjectFactory;
```

Comment:
No changes required to this class.

Reply:
My point on this one is that the ONLY changes you have to this class are import reordering.  Changes to this class are completely irrelevant to the pull request and can be reverted.